test/uiSchema_test.js

```diff
@@ -342,6 +342,46 @@ describe("uiSchema", () => {
         expect(node.querySelectorAll(".custom option")).to.have.length.of(2);
       });
     });
+    describe("enum fields disbaled options", () => {
+      const schema = {
+        type: "object",
+        properties: {
+          field: {
+            type: "string",
+            enum: ["foo", "bar"],
+          },
+        },
+      };
+      const CustomWidget = props => {
+        const { value, options } = props;
+        const { enumOptions, className, enumDisabled = [] } = options;
+        return (
+          <select className={className}>
+            {enumOptions.map(({ label, value }, i) => (
+              <option key={i} disabled={enumDisabled.indexOf(value) != -1}>
+                {value}
+              </option>
+            ))}
+          </select>
+        );
+      };
+      const uiSchema = {
+        field: {
+          "ui:widget": CustomWidget,
```

Comment:
I don't think you need a custom widget in this test, since you're updating the behavior of the `SelectWidget`, right?

Reply:
I thought it was better to add some tests since my last PR had the same feature where the reviewer asked me for some basic test case